customer-service/omnichannel/smart-assist-bot/README.md

```diff
@@ -1,14 +1,158 @@
 # Build a custom smart assist bot
 
+### Getting Started
+This sample code contains custom smart bot implementation for suggesting KB articles and for Natural Language Understanding using [LUIS](https://www.luis.ai/home).
```

Comment:
This sample code contains custom smart bot implementation for suggesting KB articles and appointment using LUIS provided by Azure Cognitive services.

Reply:
Done!